pkg/apis/maistra/v1/servicemeshmember_types.go

```diff
@@ -90,6 +90,16 @@ type ServiceMeshMemberStatus struct {
 	// generation of the object.
 	ObservedGeneration int64 `json:"observedGeneration"`
 
+	// The generation of the ServiceMeshControlPlane object observed by the
+	// controller during the most recent reconciliation of this
+	// ServiceMeshMember.
+	ServiceMeshGeneration int64 `json:"meshGeneration,omitempty"`    // TODO: do we need this field at all?
+
+	// The reconciled version of the ServiceMeshControlPlane object observed by
+	// the controller during the most recent reconciliation of this
+	// ServiceMeshMember.
+	ServiceMeshReconciledVersion string `json:"meshReconciledVersion,omitempty"`	// TODO: do we need this field at all?
```

Comment:
what's the version stored here? the resourceVersion or the control plane version?

Reply:
This is the operatorVersion and the SMCP generation. The only reason I introduced this field here is because we have the same field in the SMMR status (and the point of this PR is to move some of the functionality from SMMR to SMM). I'm not sure if users actually need it (or the one in SMMR), but they might use it to tell whether the member namespace has been updated to the latest SMCP generation. 

I'd actually like to remove this and the `ServiceMeshGeneration` field from the ServiceMeshMember and not add it until it's actually needed. Neither field is currently used by the operator.